lesson03.find-a-pair/index.html

```diff
@@ -1,22 +1,358 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <title>Find A Pair</title>
+    <title>Memory Cards Game</title>
     <meta charset="utf-8">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <style>
         body {
+            background-color: #FFF;
+            font-family: Arial, Helvetica, sans-serif;
+            margin: 0 auto;
+        }
+
+        header {
+            background-color: #469095;
+            position: fixed;
+            width: 100%;
+            z-index: 100;
+        }
+
+        h1 {
+            color: #FFF;
+            font-size: 30px;
+            font-weight: bold;
+            text-align: center;
+            text-transform: uppercase;
+            margin: 20px;
+        }
 
+        .btn {
+            font-size: 18px;
+            position: absolute;
+        }
+
+        .container {
+            position: relative;
+            top: 100px;
+            text-align: center;
+            max-width: 600px;
+            margin: 0 auto;
+            display: flex;
+            flex-direction: column;
+        }
+
+        .row {
+            display: flex;
+            justify-content: center;
+        }
+
+        .cell {
+            width: 100px;
+            height: 100px;
+            border: 3px solid #469095;
+            border-radius: 15px;
+        }
+
+        .unmatched {
+            background-image: none;
+            background-color: #f5faec;
+        }
+
+        .matched {
+            background-image: none;
+            background-color: #FFF;
+            border-color: #FFF;
+        }
+
+        .dog {
+            background: url('dog.jpg') no-repeat center;
+        }
+
+        .cat {
+            background: url('cat.jpg') no-repeat center;
+        }
+
+        .mouse {
+            background: url('mouse.jpg') no-repeat center;
+        }
+
+        .rabbit {
+            background: url('rabbit.jpg') no-repeat center;
+        }
+
+        .lion {
+            background: url('lion.jpg') no-repeat center;
+        }
+
+        .spider {
+            background: url('spider.jpg') no-repeat center;
+        }
+
+        .buffalo {
+            background: url('buffalo.jpg') no-repeat center;
+        }
+
+        .shark {
+            background: url('shark.jpg') no-repeat center;
         }
     </style>
 </head>
+
 <body>
     <header>
-        <h1>>Find A Pair</h1>
+        <h1>Memory Cards Game</h1>
     </header>
     <main>
-        Content Here
+        <div class="container">
+            <button class="btn" onclick="game.start();">Start</button>
+            <div class="row">
+                <div class="cell unmatched" index="0"></div>
```

Comment:
Can you make it more efficient by dynamic DOM manipulation? (hint - What would you do for 100?)